docs/index.md

```diff
@@ -1,5 +1,89 @@
+    - **`-w` or `--quantification_window_size` or `--window_around_sgrna`**: (default: 1) For base editors, we 
+        recommend using <font color="blue"> 20 </font>. Please bear in mind that a wider window results in more reads 
+        with sequencing errors being classified as edited. For knockout experiments, the recommendation is to use the 
+        default value.
+    - **`-wc` or `--quantification_window_center` or `--cleavage_offset`**: (default: -3) For base editors, we 
+        recommend using the center of the guide, i.e. <font color="blue"> -10 </font>. The default value, i.e. the cut 
+        position, can be used for SpCas9 knockout experiments. This parameter can be varied accordingly based on the 
+        cut position of the nuclease used. 
+    - **`--exclude_bp_from_left`**: Due to the presence of stagger sequences in our vectors, reads need to be 
+        trimmed and the provided trimming adapters cannot be used. To trim the reads on the left end this parameter can 
+        be toggled appropriately depending on the position of the sgRNA in the amplicon sequence.
+    - **`--exclude_bp_from_right`**: This argument can be used to trim reads that have low sequencing quality at the ends.
+
+    **Sample batch file shown below:**    
+
+    **name**|**fastq_r1**|**amplicon_seq**|**guide_seq**|**w**|**wc**|**exclude_bp_from_left**|**exclude_bp_from_right**|**plot_window_size**|
+    :-----:|:-----:|:-----:|:-----:|:-----:|:-----:|:-----:|:-----:|:-----:|
+    BEV_417_F3_R2|A549_RDA569_RepA_D7_Dropout_|GCAGAAAGTCAGTCCCATGGAATTTTCGCTTCCCACAGGTCTCTGCTAGGGGGCTGGGGTTGGGGTGGGGGTGGTGGGCCTGCCCTTCCAATGGATCCACTCACAGTTTCCATAGGTCTGAAAATGTTTCCTGACTCAGAGGGGGCTCGACGCTAGGATCTGACTGCGGCTCCTCCATGGCAGTGACCCGGAAGGCAGTCTGGCTGCTGCAAGAGGAAAAGTGGGGATCCA|GCTCCTCCATGGCAGTGACC|20|-10|8|8|40|
+
+
+**Step 4: Running CRISPResso2**
+
+Enter your email address for the notification email, then click the button that says "Run CRISPResso2." Similar to PoolQ, you will get a notification if your BEV run dies or succeeds. 
+   
+    **Few additional tips/tricks:**
+
+    + For experiments with both knockout and base editor samples, the quantification window size and window center 
+    parameters can be toggled appropriately to run all samples together.
+    + --skip_failed is a good argument to use while running CRISPResso2 in batch mode. This argument makes sure the job does
+    not quit if one sample fails. The failed sample can be identified in the RUNNING_LOG file.
+    + One common error you may encounter is **"CRISPResso batch #x was killed by your system. Please decrease the 
+    number of processes (-p) and run again."** Using  the `--suppress-plots` or `--suppress-report` arguments might help fix 
+    this error.  If not, please try increasing the number of resources such as "CPUs" and "Memory" in your Docker 
+    preferences.
+    + If you are encountering an error where CRISPResso2 is not able to align any reads to your reference sequence, 
```

Comment:
Note that users should download the CRISPResso output from BEV, and look through the RUNNING_LOG file in the "crispresso_output" folder to make sure all their samples were aligned and analyzed properly. If not, they should consider toggling a few parameters one of them being --default_min_aln_score.